pandas/tests/frame/test_analytics.py

```diff
@@ -2260,3 +2260,14 @@ def test_series_nat_conversion(self):
         df.rank()
         result = df
         tm.assert_frame_equal(result, expected)
+
+    def test_multiindex_column_lookup(self):
+        df = pd.DataFrame(
+            columns=pd.MultiIndex.from_product([['x'], ['a', 'b']]),
+            data=[[0.33, 0.13], [0.86, 0.25], [0.25, 0.70], [0.85, 0.91]])
+        pd.util.testing.assert_frame_equal(
```

Comment:
just use ``tm.assert_frame_equal`` its already imported.
use 

result = 
expected = 
tm.assert_frame_equal(result, expected)

put a comment for each case (if needed, may not be here)